src/utils/docsFinder.ts

```diff
@@ -34,7 +34,9 @@ export async function findDocumentation(
     case "collection":
       files = await globArray([
         `${dir}/ansible_collections/*/*/plugins/modules/*.py`,
+        `${dir}/ansible_collections/*/*/plugins/modules/**/*.py`,
         `!${dir}/ansible_collections/*/*/plugins/modules/_*.py`,
+        `!${dir}/ansible_collections/*/*/plugins/modules/**/_*.py`,
```

Comment:
to avoid dupes (aliases) i would filter out symlinks

Reply:
also, module docs can be in more than .py extension (.yml and .yaml)

Reply:
> to avoid dupes (aliases) i would filter out symlinks

According to the `glob` package documentation (https://www.npmjs.com/package/glob):
`**` If a "globstar" is alone in a path portion, then it matches zero or more directories and subdirectories searching for matches. It does not crawl symlinked directories.

> also, module docs can be in more than .py extension (.yml and .yaml)

Can you provide an example ?
If I exec `find -regex '.*\.ya?ml' | grep 'plugins/modules'` in `ansible_collections` directory of ansible pip package (6.4.0 version), the output is empty.

Reply:
symlinked dirs are not normally used as an alias (though they could work), symlinked files are

Reply:
I had a commit to filter symlinked files https://github.com/ansible/ansible-language-server/pull/449/commits/e61324cd6502ea51a4c2ae641d5de037c686fcb8

But for yaml/yml extensions, I don't know what to do.
I tested in a virtualenv with ansible 7.0.0a1 and ansible-core 2.14.0b2: the output of `find -regex '.*\.ya?ml' | grep 'plugins/modules'` is still empty.

Reply:
yet you can see a bunch of YAML files live there now https://github.com/ansible/ansible/tree/devel/lib/ansible/plugins/filter

Reply:
it will still take time for collections to adopt this, since it is not even released yet

Reply:
Thanks for the answer.
But I don't think that supporting future developments is in the scope of this PR. The goal of this request is just to fix a current issue that affect users (including me).

Reply:
just FYI since 2.14 is about to be released